src/pages/achievement/RankingTab.tsx

```
@@ -35,24 +36,34 @@ function RankingTab() {
+      if (isFirstFocus.current) {
+        isFirstFocus.current = false;
+        return;
       }
-    };
+      fetchRanking();
+    }, [fetchRanking])
+  );
 
+  useEffect(() => {
     fetchRanking();
   }, [rankingFilter]);
```

ESLint

[error] React Hook useEffect has a missing dependency: 'fetchRanking'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)